### PR TITLE
Add guide for audio usage.

### PR DESCRIPTION
**Pull request by btheosam**
Add a new document HowToUseAudio.md on how to use audio.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToUseAudio.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	 * driver.
+	*/
+	pcm = pcm_decode_initialize(alc5658);
+
+	/*Embed PCM device witin audio device */
+	ret = audio_register(devname, pcm);
+```
+
+Example: artik053_alc5658.c  
+
```
Comment (btheosam) [inline docs/HowToUseAudio.md]: After this, please add how to develop app on top of Tinyalsa APIs.

File: docs/HowToUseAudio.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+## Audio Device Registration
+
+After setting up audio codec, the codec's lowerhalf is wrapped around a PCM decoder device  
+and which is again wrapped around a audio device.  
+
+Psuedocode
+
+```
+	/* No we can embed the ALC5658/I2C/I2S conglomerate into a PCM decoder
```
Comment (btheosam) [inline docs/HowToUseAudio.md]: Fix the comment, remove alc5658 and make it a generic. Where does the alc5658 comes? need more clarity.

Comment (Taejun-Kwon) [reply docs/HowToUseAudio.md]: I think you can suggest alc5658 as a example.
Also code location will be helpful.

Comment (btheosam) [reply docs/HowToUseAudio.md]: Okay, then this shall be left.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**